site/viamguides/src/smart-snack-dispenser/smart-snack-dispenser.md

```diff
@@ -0,0 +1,444 @@
+   ![select video path](assets/configureCamera.webp)
+1. Click **Save** in the top right. This may take a moment to apply your configuration changes.
+1. At the bottom of the `camera-1` panel, expand the **TEST** section to ensure you have configured the camera properly and see a video feed.
+   ![test camera](assets/testCamera.webp)
+
+> aside negative
+> **TROUBLESHOOTING**: Double check the wiring on your components. Also check under the **LOGS** tab to see what might be going wrong.
+
+With your hardware components confirmed working, grab your snacks to start gathering data to train a custom computer vision model in the next step.
+
+<!-- ------------------------ -->
+## Collect data and train the model
+Duration: 30
+
+To get started with capturing training data for the machine learning model used to automate the smart snack dispenser, fill the top container (hopper) of the dispenser with your snack of choice that you want to model to be able to identify.
+
+### Add the data manager
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `data management`.
+1. **Create** a new [Data Management service](https://docs.viam.com/services/data/) called `data_manager-1`.
+   ![select data management](./assets/addDataManager.webp)
+1. Notice adding this service adds the data manager service called `data_manager-1`. It will default to the "Capturing" and "Syncing" toggles as enabled.
+   ![view default configuration for data manager](./assets/configureDataManager.webp)
+1. Click **Save** in the top right. This may take a moment to apply your configuration changes.
+
+### Capture images from the webcam
+
+1. In the `camera-1` panel, click the "+ Add method" button under the **Data capture** section.
+1. For the "Method" dropdown, select "ReadImage", enter "0.25" in the "Frequency (hz)" field to capture an image every 4 seconds.
+1. Make sure the data capture toggle is "On".
+   ![add data capture method](./assets/configureCameraCapture.webp)
+1. Click **Save** in the top right. This may take a moment to apply your configuration changes.
+1. At the bottom of the `camera-1` panel, expand the **TEST** section to view the images as they're being captured.
+   ![test camera](assets/testCamera.webp)
+1. Click the "Toggle picture-in-picture" button to create a separate window of the camera view.
+1. Go to the `motor-1` panel and expand the **TEST** section at the bottom. Use the controls to turn the motor and dispense the snacks into the bowl.
+   ![dispense snacks with motor](assets/enableMotorControl.webp)
+1. Once the bowl is full, empty the bowl back into the container and dispense more snacks. Go through this cycle a few times to ensure you have enough image data to train an accurate model.
+1. When you feel you have enough images, you can close the picture-in-picture window of the camera and turn off data capture in the `camera-1` panel.
```

Comment:
```suggestion
1. When you feel you have enough images (with a minimum of 25), you can close the picture-in-picture window of the camera and turn off data capture in the `camera-1` panel.
```

Adding a minimum number of images to ensure a greater chance of their model working. Feel free to change the number.